### PR TITLE
accounts: localized site name for user email

### PR DESCRIPTION
**Pull request by jmartinm**
* BETTER When contacting users, use the localized site name.
  (addresses #3273)

### CODE REVIEW

**Comment by jmartinm** [comment]
This allows to still leave the ```SITE_NAME == main collection name``` and have a friendly site name to be used when communicating with users.

For instance, our config looks like:

```python
CFG_SITE_LANG = u"en"
CFG_SITE_LANGS = ['en', ]

# CFG_SITE_NAME and main collection name should be the same for empty search
# to work
CFG_SITE_NAME = u"HEP"

langs = {}
for lang in CFG_SITE_LANGS:
    langs[lang] = u"INSPIRE Labs"
CFG_SITE_NAME_INTL = langs

```

**Comment by jirikuncar** [comment]
:+1: :shipit:

File: invenio/modules/accounts/helpers.py [hunk 1]
```diff
@@ -59,6 +59,6 @@ def send_account_activation_email(user):
         cfg.get('CFG_SITE_SUPPORT_EMAIL'),
         user.email,
         _("Account registration at %(sitename)s",
-          sitename=cfg['CFG_SITE_NAME']),
+          sitename=cfg["CFG_SITE_NAME_INTL"].get(g.ln, cfg['CFG_SITE_NAME'])),
```
Comment (jirikuncar) [inline invenio/modules/accounts/helpers.py]: `s/g.ln/getattr(g, 'ln', cfg['CFG_SITE_LANG'])/`

**Comment by jmartinm** [comment]
Code updated and ```Reviewed-by``` added

File: invenio/modules/accounts/helpers.py (hunk 1 above)
Comment (jirikuncar) [inline invenio/modules/accounts/helpers.py]: @jmartinm ping

**Pull request merged**